### ISSUE TITLE
decide on crediting

### ISSUE THREAD

**Issue by harlo** [freedomofpress/encryption-works#186]
...as @ageis notes:

We need to decide on crediting. @harlo and @garrettr are credited as editors, but if this gets merged I'm technically one too. @trevortimm edited as well, but wasn't credited. It's imported to credit Micah Lee and Tommy Collison at the least. We could add me, or say "others" or FPF.

**Comment by conorsch** [comment]
We're already pretty verbose with attribution. Here's what's proposed in the new version:

> Originally written by Micah Lee. Updated version written by Tommy Collison and edited by Harlo Holmes and Garrett Robinson.

That's four names over two sentences. Does our target audience care about that lineage? We could name Micah and Tommy, then collapse everyone else into the FPF umbrella, e.g.

> Originally written by Micah Lee. Updated version written by Tommy Collison and edited by [FPF staff members](https://freedom.press/about/staff/).

However, even using the FPF umbrella as a catch-all is tricky, since we have contributors who aren't FPF:

```
$ git ls-files -z | \
    xargs -0n1 git blame -w | \
    perl -n -e '/^.*?\((.*?)\s+[\d]{4}/; print $1,"\n"' | \
    sort -f | uniq -c | sort -nr
  10041 Micah Lee
   4009 tommycollison
   2203 Garrett Robinson
   1646 Tommy Collison
     98 Kevin M. Gallagher
     20 Richard Littauer
      7 harlo
      5 William Budington
      1 securepress

```
If the document is intended to be community maintained, we should prepare to abandon naming every contributor without regard of the size of contributions. One of the reason we're using git is because we get attribution tracking for free. There's no question that Micah and Tommy have done the lion's share of work on the document, and splitting hairs to give "editing" credit based on the type or magnitude of contributions only detracts from the message of the document.

Let's just fold everyone who's not Micah and Tommy into "others", or "numerous contributors." Further down the line, we can reevaluate, and perhaps even remove those names, replacing with an intentionally vague and nonproprietary "maintained by FPF."

**Comment by garrettr** [comment]
Since we're doing everything on GIthub, the distinction between author and editor is IMHO even fuzzier than it usually is - for example, recent work by "editors" like @conorsch and @ageis have resulted in them filing pull requests to fix the issues they identified.

I'd advocate for Micah and Tommy being credited by authors in the document, and everybody else being credited in a CONTRIBUTORS file as is commonly used by open source projects (e.g. [Go's CONTRIBUTORS](https://www.youtube.com/watch?v=A2_snSkpULQ)).

**Comment by ageis** [comment]
@garrettr +1 I second that and I suspect the rest of us are on board with that idea.

**Comment by conorsch** [comment]
@garrettr @ageis Sounds fine to me!

**Comment by conorsch** [comment]
Resolved by #188 and #191. Tracking attribution for README in #193.
